src/assets/style/module/controls/_controls.less

```diff
@@ -3,22 +3,36 @@
     display: inline-block;
 }
 
+#controls > .github-logo{
+    width: auto;
+}
+
+.github-logo {
+    img {
```

Comment:
Instead of using a descendant selector, let's add `.github-logo-img` and add this directly to the image tag in the markup.  I'm not sure if you're using the `.github-logo` class anywhere outside of css but, if not, we can actually remove it.